packages/dd-trace/test/appsec/response_blocking.spec.js

```diff
@@ -55,6 +56,8 @@ describe('HTTP Response Blocking', () => {
         rules: path.join(__dirname, 'response_blocking_rules.json')
       }
     }))
+
+    disableLfi()
```

Comment:
same here, I can imagine that without this disableLfi the tests are failing, but I don't understand why, write a comment here please

Reply:
as rasp is enabled by default, the test execution triggers some lfi checks executing `waf.run` and changing the expectation of the test.

But, instead of calling `disableLfi()` I have changed it to disable rasp with a config flag.